reactivesocket-client/src/main/java/io/reactivesocket/client/LoadBalancer.java

```diff
@@ -867,6 +867,7 @@ public String toString() {
                 + " quantile-high:" + higherQuantile.estimation()
                 + " inter-arrival:" + interArrivalTime.value()
                 + " duration/pending:" + (pending == 0 ? 0 : (double)duration / pending)
+                + " pending:" + pending
```

Comment:
Default `toString()` generator for Intellij? Although it uses `StringBuilder` I think. So, it becomes 2 steps, generate `toString()` and then accept intention of using string concatenation 😄